presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -3251,115 +3252,96 @@ public void testShowCreateTable()
         assertEquals(getOnlyElement(actualResult.getOnlyColumnAsSet()), createTableSql);
     }
 
-    @Test
-    public void testCreateExternalTable()
+    /**
```

Comment:
The `Also, add a separate test for the default field delimiter` should be extracted to separate commit.